packages/kbn-legacy-logging/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@kbn/legacy-logging",
+  "version": "1.0.0",
+  "private": true,
```

Comment:
Could you add a Readme that this package is a temporary workaround until we migrate logging to the KP?

Reply:
I was more planning to keep it there until we totally get rid of the legacy logging system in 8.0 tbh. It will be way easier to delete it with all the code being centralized in this package. Was that what you meant?